addons/web/static/src/model/relational_model/dynamic_list.js

```diff
@@ -250,6 +250,7 @@ export class DynamicList extends DataPoint {
             if (resIds.length > copiedRecords.length) {
                 this.model.notification.add(_t("Some records could not be duplicated"));
             }
+            await this.model.hooks.onDuplicatedMulti(resIds, copiedRecords);
```

Comment:
I would have a single hook `onRecordsDuplicated` which always takes a list of ids (1 item in form view) in argument

Reply:
or two lists